install/kubernetes/cilium/values.yaml

```diff
@@ -27,6 +27,10 @@ preflight:
   # cluster with the right schema.
   validateCNPs: true
 
+# Include the Hubble-ui config
```

Comment:
I _believe_ this has to be part of the global section so you can read it out in the `config` chart (to conditionally modify `hubble-listener-address`).

Reply:
Will address this in the next push.

Reply:
Addressed review comments.